chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -158,9 +158,14 @@ The native window referred to by pname:pCreateInfo->surface must:
 not become associated with a non-Vulkan graphics API surface before
 all associated Vulkan swapchains have been destroyed.
 
-Like core functions, several WSI functions, including
-fname:vkCreateSwapchainKHR return ename:VK_ERROR_DEVICE_LOST if the logical
-device was lost.
+fname:vkCreateSwapchainKHR can return ename:VK_ERROR_DEVICE_LOST if the logical
```

Comment:
"Can" is keyword. Avoid colloquial usage.

Reply:
ACK. I'll try to get to this in the next week. Way behind.

Reply:
replaced with "will"